api/src/test/java/au/org/aodn/nrmn/restapi/service/SpreadSheetServiceIT.java

```diff
@@ -125,6 +124,18 @@ public void shortInvertsRowShouldBeInvalid() throws Exception {
         }
         assertTrue(StringUtils.isNotEmpty(error));
     }
+    /**
+     * Excel row2 must be blank according to the import file spec, if not blank throw exception
+     */
+    @Test
+    void rejectFileWhenRow2IsNotEmpty() {
+        Exception e = assertThrows(Exception.class, () -> {
+            FileSystemResource file3 = new FileSystemResource("src/test/resources/sheets/row2NotEmpty.xlsx");
+            ParsedSheet parsedSheet = sheetService
```

Comment:
> Getting this error when trying to reingest an exported sheet: `Row 1 missing headers: 50. Row 1 has unexpected headers: UNKNOWN`.
> 
> Please also move the Field enum into a separate file and rename it something like `SurveyField`.

The missing headers error is due to a mismatch type in constant.js, I fixed it and make a conversion in export function so that the header will always be type 'String' 

SurveyField refactored and become a standalone file